l10n/en/banners/fundraising.ftl

```diff
@@ -4,25 +4,16 @@
 
 ### URL: https://www-dev.allizom.org/
 
-banner-fundraising-title-v2 = Help { -brand-name-mozilla } fight for a better internet this holiday season.
+banner-fundraising-giving-tuesday-title = Donate to { -brand-name-mozilla } for Giving Tuesday
```

Comment:
I wonder if "Giving Tuesday" is too American... I know "Black Friday" has taken over the world, because western capitalism, but I don't know if Giving Tuesday has become known internationally. Does this warrant a comment for localizers? Maybe a question for Peiying?